GameData/RP-0/Strategies/Leaders/LeadersEngineers.cfg

```diff
@@ -389,3 +389,59 @@ STRATEGY
 		}
 	}
 }
+
+STRATEGY
+{
+	name = leaderValCleaverEngineer
+	RP0conf = True
+	title = Arthur Valentine Cleaver
+	desc = Arthur "Val Cleaver" is a British Aerospace Engineer working for de Havilland in the early 1950s, responsible for overseeing the development of the early British rocket plane engines, "Spectre" and "Sprite". Later he worked to develop upgraded variants of the "Spectre" engine, that went on to power the Blue Steel missile.  In 1956, Val Cleaver hands in his notice to de Havilland, and takes a job with Rolls-Royce as their Chief Rocket Propulsion Engineer. Under his guidance the development of the RZ.2 and RZ.20 takes place, built to power the Blue Streak missile. Considered highly advanced engines at the time, it later goes on to power the first stage of Europa 1. Arthur Val Cleaver is awarded an Order of the British Empire, a prestigious medal for prominent national achievements.
+	department = Engineering
+	icon = RP-0/Strategies/Leaders/ValCleaver
+	iconDepartment = RP-0/Strategies/Leaders/ValCleaver
+
+
+	groupTag = Engineering,leaderValCleaver
+
+	removeOnDeactivate = True
+	removeOnDeactivateTag = leaderValCleaver
+  
+	minDuration = 2592000 // 30 days
+	maxDuration = 315576000 // 10 years
+	reactivateCooldown = 31557600 // 1 year
+	
+	EFFECT
+	{
+		name = ResearchRateModifier
+		effectTitle = Rolls-RoyceDidRocketsGood!
```

Comment:
effectTitle is the title shown ingame. It should be a human-readable phrase, yours seem to lack spaces.